code/datums/status_effects/debuffs.dm

```diff
@@ -824,3 +824,101 @@
+	switch(current_fake_disease)
+		if(FAKE_COLD)
+			fake_msg = list(
+						list("<span class='danger'>Your throat feels sore.</span>", "<span class='danger'>Mucous runs down the back of your throat.</span>"),
+						list("<span class='danger'>Your muscles ache.</span>", "<span class='danger'>Your stomach hurts.</span>"),
+						list("<span class='danger'>Your muscles ache.</span>", "<span class='danger'>Your stomach hurts.</span>")
+			)
+			fake_emote = list(
+						list("sneeze", "cough"),
+						list("sneeze", "cough"),
+						list("sneeze", "cough")
+			)
+		if(FAKE_FOOD_POISONING)
+			fake_msg = list(
+						list("<span class='danger'>Your stomach feels weird.</span>", "<span class='danger'>You feel queasy.</span>"),
+						list("<span class='danger'>Your stomach aches.</span>", "<span class='danger'>You feel nauseous.</span>"),
+						list("<span class='danger'>Your stomach hurts.</span>", "<span class='danger'>You feel sick.</span>")
+			)
+			fake_emote = list(
+						list(null),
+						list("groan"),
+						list("groan", "moan")
+			)
+		if(FAKE_RETRO_VIRUS)
+			fake_msg = list(
+						list("<span class='danger'>Your head hurts.</span>", "You feel a tingling sensation in your chest.", "<span class='danger'>You feel angry.</span>"),
+						list("<span class='danger'>Your skin feels loose.</span>", "You feel very strange.", "<span class='danger'>You feel a stabbing pain in your head!</span>", "<span class='danger'>Your stomach churns.</span>"),
+						list("<span class='danger'>Your entire body vibrates.</span>")
+			)
+		else
+			fake_msg = list(
+						list("<span class='danger'>Your chest hurts.</span>", "<span class='danger'>Your stomach violently rumbles!</span>", "<span class='danger'>You feel a cold sweat form.</span>"),
+						list("<span class='danger'>You feel a sharp pain from your lower chest!</span>", "<span class='danger'>You feel air escape from your lungs painfully.</span>"),
+						list("<span class='danger'>You feel uncomfortably hot...</span>", "<span class='danger'>You feel like unzipping your jumpsuit</span>", "<span class='danger'>You feel like taking off some clothes...</span>")
+			)
+			fake_emote = list(
+						list("cough"),
+						list("gasp"),
+						list(null)
+			)
+	. = ..()
+
+/datum/status_effect/fake_virus/tick()
+	var/selected_fake_msg
+	var/selected_fake_emote
+	switch(msg_stage)
+		if(0 to 300)
+			if(prob(1)) // First stage starts slow, stage 2 and 3 trigger fake msgs/emotes twice as often
+				if(prob(50) || !listgetindex(fake_emote, 1)) // 50% chance to trigger either a msg or emote, 100% if it doesnt have an emote
+					selected_fake_msg = pick(fake_msg[1])
+				else
+					selected_fake_emote = pick(fake_emote[1])
+		if(301 to 600)
+			if(prob(2))
+				if(prob(50) || !listgetindex(fake_emote, 2))
+					selected_fake_msg = pick(fake_msg[2])
+				else
+					selected_fake_emote = pick(fake_emote[2])
+		else
+			if(prob(2))
+				if(prob(50) || !!listgetindex(fake_emote, 3))
```

Comment:
```suggestion
				if(prob(50) || !listgetindex(fake_emote, 3))
```
One too many `!` here. The check shouldn't be needed though since you supply 3 lists for each fake virus type.

Reply:
Without this check it wouldnt 50% of the time do nothing if the fake virus has no fake_emotes to feed to it?

Reply:
Each instance of `fake_emote` and `fake_msg` has exactly 3 lists in it. So getting the first, second and third item in each list will always be possible

Reply:
It will always be possible but the emote() and to_chat are going to trigger with an selected_fake_msg/emote that is empty, in practice cutting the amount of messages the player receives in half when the stage he is on has msg but no emotes.

Reply:
Actually..think i got it now.

Reply:
Ah yeah I see. Since `list()` will evaluate to `FALSE` if you put it into an if. Right that works. Can you change the code to use `!length(fake_emote[3])` instead? Since you know the third element will exist. You just don't know if that element has items itself or not

Reply:
Yep, was on my way to try that.